x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/LongVectorFixedBuilder.java

```diff
@@ -38,6 +40,12 @@ public LongVectorFixedBuilder appendLong(long value) {
         return this;
     }
 
+    @Override
+    public LongVectorFixedBuilder appendLong(long value, int idx) {
```

Comment:
I hate to ask this, but could you reverse the argument order? That way it *looks* a little more like array assignment. It's a lot of changing things, but hopefully they are all generated.

Reply:
Yes, most of the code is generated, it should be easy ;-)

Reply:
Thank you!